index.html

```diff
@@ -0,0 +1,2 @@
+<iframe src="editor/svg-editor.html?extensions="
+    width="100%" height="100%"></iframe>
```

Comment:
I don't know that we want this for all projects. But feel free to add `index.html` to `.gitignore` as a kind of placeholder.

Reply:
No, I had just used this to run the application locally.

Reply:
Ok, so if you can remove this from the PR then.

Reply:
Ok!